docs/pages/machine-id/guides/applications.mdx

```diff
@@ -0,0 +1,281 @@
+---
+title: Machine ID with Application Access
+description: Accessing web applications with Machine ID
+---
+
+In this guide, we will demonstrate how to configure an automated service to use Machine ID to access a web application protected by Teleport.
+
+With Machine ID, Teleport issues short-lived certificates, tied to a machine
+identity, that can be rotated, audited, and managed with the same access
+controls that Teleport provides for human users.
+
+## Prerequisites
+
+<ScopedBlock scope={["cloud"]}>
+You will need Teleport Cloud 10.1.0 or later.
+</ScopedBlock>
+<ScopedBlock scope={["oss","enterprise"]}>
+You will need to be running Teleport version 10.1.0 or later.
+</ScopedBlock>
+
+If you have not already connected your application to Teleport, follow
+the [Application Access Getting Started Guide](../../application-access/getting-started.mdx).
+
+If you're not already familiar with Machine ID, follow the
+[Getting Started Guide](../getting-started.mdx) to familiarize yourself with
+Machine ID. You'll also need `tctl` access to initially configure the bot.
+
+(!docs/pages/includes/tctl.mdx!)
+
+Lastly, ensure the `tbot` binary is installed on your Machine ID client system.
```

Comment:
I think we should be more specific about what the client system entails. Is this a VM with access to the Auth Service or Proxy Service and the application you want Machine ID to access? Would a user be able to follow this guide with the `tbot` binary installed on their local machine?

Reply:
I've reworked this to hopefully better explain what a client system is. I didn't say it explicitly, but it is perfectly fine to run it on your local machine to try things out. I wasn't sure if that would create more confusion, though.